### ISSUE TITLE
HTML report: small screens

### ISSUE THREAD

**Issue by jernst** [fediverse-devnet/feditest#419]
From Accessibility report:

> The website does not work on a small screen. That is clear when making the screen smaller. That is not WCA proof, but reasonable. Either way my recommendation is that you look for a way to make it phone proof. If you want to take a quick look at the bug while you are not on a laptop it is not possible.

**Comment by jernst** [comment]
This is out of scope, at least for now. We have more important things to do first.
